sync/src/synchronizer/mod.rs

```diff
@@ -344,13 +344,21 @@ impl Synchronizer {
         }
     }
 
-    fn find_blocks_to_fetch(&self, nc: &dyn CKBProtocolContext) {
+    fn find_blocks_to_fetch(&self, nc: &dyn CKBProtocolContext, is_ibd: bool) {
         let peers: Vec<PeerIndex> = {
             self.peers()
                 .state
                 .read()
                 .iter()
-                .filter(|(_, state)| state.sync_started)
+                .filter(|(_, state)| {
+                    if is_ibd {
+                        state.peer_flags.is_outbound
```

Comment:
Why can not fetch blocks from inbound peers?

Reply:
> Why can not fetch blocks from inbound peers?

`32 outbound peer, sync from 0 to 494467，took 110 minutes，average speed 4495/min` must fix this first

Reply:
I think the reason is that outbounds are chosen randomly, which are consider safer than inbounds.